### PR TITLE
feat: Bridged usdc update + batching EthV2 RF updates, AvaxV2 RF upates, PolV2 BR updates

### PR DESCRIPTION
**Pull request by Luigy-Lemon**
- [Snapshot for USDCe updates](https://snapshot.org/#/aave.eth/proposal/0x9cc7906f04f45cebeaa48a05ed281f49da00d89c4dd988a968272fa179f14d06)
- [Discussion for USDCe updates](https://governance.aave.com/t/arfc-increase-bridged-usdc-reserve-factor-across-all-deployments/17787)
- [Discussion for Ethereum V2 Reserve Factor Updates](https://governance.aave.com/t/arfc-ethereum-v2-reserve-factor-adjustment/16764/13?u=luigy)
- [Snapshot for Ethereum V2 Reserve Factor Updates](https://snapshot.org/#/aave.eth/proposal/0x26a03c08359c340f63b78b0c3e96d37aa0adeda65814643b0886d4719048ea7e)
- [Discussion for Avalanche V2 Reserve Factor Updates](https://governance.aave.com/t/arfc-avalanche-v2-reserve-factor-adjustment/17040/8?u=luigy)
- [Snapshot for Avalanche V2 Reserve Factor Updates](https://snapshot.org/#/aave.eth/proposal/0x770ff4e02634c77aaa09952345551168920f7878b32ab03fcef92763a5fb70ab)
- [Discussion for Polygon V2 Borrow Rate Updates](https://governance.aave.com/t/arfc-polygon-v2-borrow-rate-adjustments/17252/8?u=luigy)
- [Snapshot for Polygon V2 Borrow Rate Updates](https://snapshot.org/#/aave.eth/proposal/0x95643085ee16eb0eaa4110a9f0ea8223009f9521e596e1a958303705a5001363)
- [Discussion for Polygon V2 RF update](https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/23?u=dd0sxx)

### CODE REVIEW

File: src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9cc7906f04f45cebeaa48a05ed281f49da00d89c4dd988a968272fa179f14d06"
+---
+
+## Simple Summary
+
+This AIP is composed of three actions:
+
+1. To start periodically increasing the Reserve Factor (RF) for Bridged USDC(USDC.e & USDbC) across Arbitrum, Optimism, Polygon and Base Aave deployments.
+2. A continuation of the proposals on Governance V3 that increases the Reserve Factor (RF) for assets on Ethereum V2 and Avalanche V2 by 5.00%, up to a maximum of 99.99%;
+3. A continuation of the proposal to adjust Polygon v2 Borrow Rate
```
[the remark this answers is not in the document]
Comment (Luigy-Lemon) [reply src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md]: done

File: src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-increase-bridged-usdc-reserve-factor-across-all-deployments/17787"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9cc7906f04f45cebeaa48a05ed281f49da00d89c4dd988a968272fa179f14d06"
+---
+
+## Simple Summary
+
+This AIP is composed of three actions:
+
+1. To start periodically increasing the Reserve Factor (RF) for Bridged USDC(USDC.e & USDbC) across Arbitrum, Optimism, Polygon and Base Aave deployments.
```
[the remark this answers is not in the document]
Comment (Luigy-Lemon) [reply src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md:12]: Not a typo :D

File: src/20240528_Multi_BridgedUSDCeUpdateRF/AaveV2Ethereum_ReserveFactorUpgrades_20240528.sol [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets, ILendingPoolConfigurator} from 'aave-address-book/AaveV2Ethereum.sol';
+
+/**
+ * @title Reserve Factor Upgrades
+ * @author karpatkey_TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x770ff4e02634c77aaa09952345551168920f7878b32ab03fcef92763a5fb70ab
+ * - Discussion: https://governance.aave.com/t/arfc-avalanche-v2-reserve-factor-adjustment/17040/4
```
Comment (sendra) [inline src/20240528_Multi_BridgedUSDCeUpdateRF/AaveV2Ethereum_ReserveFactorUpgrades_20240528.sol:11]: should provably be a link to the ethereum rf adjustment forum url

File: src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+3. A continuation of the [proposal](https://governance.aave.com/t/arfc-polygon-v2-borrow-rate-adjustments/17252) to adjust Polygon v2 Borrow Rate.
+
+## Motivation
+
+Presently, Bridged USDC (USDC.e & USDbC) competes with native USDC on the listed markets. By gradually increasing the RF for Bridged USDC(USDC.e & USDbC), the deposit rate on these markets will become less attractive over time. Similar to other proposals, this action is expected to encourage users to switch to native USDC on the respective market.
+
+Upon implementing this proposal, a subsequent AIP will be submitted that increases the RF by 5.00% up to a maximum of 99.99% every 2 weeks, subject to market conditions. The RF amendments will be incorporated into the fortnightly RF and Borrow Rate adjustment AIP to reduce voting overhead.
+
+This AIP will also reduce deposit yield for assets on Ethereum & Avalanche V2 by increasing the RF. With this upgrade being passed, users will be further encouraged to migrate from Ethereum V2 to V3.
```
Comment (sendra) [inline src/20240528_Multi_BridgedUSDCeUpdateRF/IncreaseUSDCeRF.md:22]: and polygon V2

**Pull request merged**